### PR TITLE
Add 'real time' and 'real-time' to terminology table (#124)

### PR DESCRIPTION
**Pull request by cwickert**
This adresses #124.

### CODE REVIEW

File: xml/docu_styleguide.terminology.xml [hunk 1]
```diff
@@ -1228,6 +1228,16 @@
       <entry>R.O., RO, read only, readonly, ro</entry>
       <entry>adjective</entry>
      </row>
+     <row>
+      <entry>real time</entry>
+      <entry>real-time</entry>
+      <entry>noun; e.g. <quote>watch in real time</quote></entry>
```
Comment (dariavladykina) [inline xml/docu_styleguide.terminology.xml:1234]:
```suggestion
      <entry>noun; as in: “watch in real time”</entry>
```

File: xml/docu_styleguide.terminology.xml [hunk 2]
```diff
@@ -1228,6 +1228,16 @@
...
+     </row>
+     <row>
+      <entry>real-time</entry>
+      <entry>real time</entry>
+      <entry>adjective, compound noun; e.g. <quote>real-time processing</quote></entry>
```
Comment (dariavladykina) [inline xml/docu_styleguide.terminology.xml:1239]:
```suggestion
      <entry>adjective, compound noun; as in: “real-time processing”</entry>
```

**Review by dariavladykina — COMMENTED** [review]
Hi Christoph, 

I've updated the entries to be consistent with "e.g." instances in our SG.

Thanks,
Daria

File: xml/docu_styleguide.terminology.xml (hunk 1 above)
Comment (cwickert) [reply xml/docu_styleguide.terminology.xml]: I've applied this manually without the quotation marks.

**Pull request merged**